src/helpers/enrich-dataset.test.ts

```diff
@@ -70,7 +70,10 @@ describe('enrichDataset function', () => {
             provenance: '',
             hubLandingPage: 'https://arcgis.com/maps/CALFIRE::DCAT_Test',
             downloadLink: 'https://arcgis.com/datasets/CALFIRE::DCAT_Test',
-            agoLandingPage: 'portal.arcgis.com/home/item.html?id=0&sublayer=0',
+            durableUrlCSV: 'https://arcgis.com/api/download/v1/item/123a/csv?layers=0',
```

Comment:
I totally get that you were just copying what existed, but I got really tripped up by the fact that these urls use `arcgis.com` instead of `hub.arcgis.com` (or `my-hub-site.com`, etc.). This is mainly because the download APIs provided by us, not ArcGIS Online. Would you mind updating for consistency?